src/models/btcPrice.js

```
@@ -0,0 +1,23 @@
+const { Entity } = require('../services/entity');
+const fetch = require('node-fetch');
+
+class BitcoinPrice extends Entity {
+  constructor() {
+    super(null, []);
+  }
+
+  async getExchangeRate() {
+    const nbuRequest = await fetch('https://bank.gov.ua/NBUStatService/v1/statdirectory/exchange?json');
+    const nbuRequestData = await nbuRequest.json();
+
+    const btcPriceRequest = await fetch('https://blockchain.info/ticker');
+    const btcPriceRequestData = await btcPriceRequest.json();
+
+    const hryvniaExchangeRate = nbuRequestData.filter((entry) => entry.cc === 'USD')[0].rate;
+    const bitcoinPriceUSD = btcPriceRequestData.USD.last;
+    const bitcoinPriceUAH = parseFloat((hryvniaExchangeRate * bitcoinPriceUSD).toFixed(2));
+    return { bitcoinPriceUAH };
+  }
+}
+
+module.exports = { BitcoinPrice };
```

A couple of fetch requests without try...catch. Are possible errors handled somewhere?

---

The possible errors are not handled... Thank you for the notice.

---

Alright) Though e is declared but never used. The linter might have warned about that :)